### PR TITLE
Revise

### PR DESCRIPTION
**Pull request by bbenligiray**
I was more referring to this kind of a thing [here](https://github.com/api3dao/data-feed-reader-example/pull/61#issuecomment-2132986996)

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -1,75 +1,45 @@
...
-You can update the proxy that your DataFeedReaderExample reads from.
-
-- Follow the [instructions](#../README.md#instructions)
-- Deploy a new proxy
-- See the command below, but use your own `NETWORK` and `PROXY` values
-
-```sh
-NETWORK=ethereum-sepolia-testnet PROXY=0x08506208E776ecbdF4cE9DB69C08Aa90A06825C0 yarn update-proxy
-```
+See https://github.com/api3dao/nodary-examples
```
Comment (bbenligiray) [inline scripts/README.md:45]: I'll revise this too

File: scripts/README.md [hunk 2]
```diff
@@ -1,75 +1,45 @@
...
 
-If you'd like to set your own OEV beneficiary address then use the `OEV_BENEFICIARY` environment variable.
+You can also refer to the [`deploy-dapi-proxy-with-oev` script](./deploy-dapi-proxy-with-oev.js) to see how the `@api3/contracts` package can be used to derive the respective `DapiProxyWithOev` address without needing to make any RPC calls.
 
-```sh
-NETWORK=ethereum-sepolia-testnet DATA_FEED_ID=0xd888b92f9d71afedd0a012622c0d1d5368fc0dc0ff1d30bb16266afcd49c2c17 OEV_BENEFICIARY=0x55Cf1079a115029a879ec3A11Ba5D453272eb61D yarn deploy-data-feed-proxy-with-oev
-```
+The OEV beneficiary of the proxy above is the API3 _manager multisig_ on that chain.
+To deploy a proxy with an arbitrary OEV beneficiary, use the command below with your `OEV_BENEFICIARY` value.
+Note that the OEV functionalities of proxies with custom beneficiaries are not supported
```
Comment (acenolaza) [inline scripts/README.md:34]: For someone just trying to read a data feed value all this OEV stuff might be a bit overwhelming. I wonder if we should add a link to some link in the docs. [This one](https://docs.api3.org/explore/dapis/OEV.html) for instance

**Review by acenolaza — APPROVED** [review]
LGTM 👍🏻 (Sorry, I thought that we still wanted to have the other deploy proxy scripts instead of mimicking exactly the market UI)

File: scripts/README.md (hunk 2 above)
Comment (bbenligiray) [reply scripts/README.md:34]: It's in the advanced section so it should be fine. I don't want to link OEV docs until they reach a stable state.

**Pull request merged**